src/get_functions.jl

```diff
@@ -0,0 +1,83 @@
+"""
+    get_origin(vtk_file)
+
+Retrieve the vector of coordinates of the origin of a uniform grid from the given [`VTKFile`](@ref) file.
+
+See also: [`VTKFile`](@ref)
+"""
+function get_origin(vtk_file)
+  # obtain dataset
+  dataset_element = get_imagedata_dataset(vtk_file)
+
+  whole_extent = get_whole_extent(vtk_file)
+  
+  # obtain the origin
+  origin = parse.(Float64, split(attribute(dataset_element, "Origin", required=true), ' '))
+
+  if (whole_extent[5:6] == [0,0]) && (origin[3] == 0)
```

Comment:
```suggestion
  if (whole_extent[5:6] == [0,0]) && (origin[3] == 0.0)
```
`origin` is a `Vector{Float64}`